apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -197,6 +197,22 @@ const radioGroupCss = css`
   .sberRadioGroup .ant-radio-button-wrapper-checked:not(.ant-radio-button-wrapper-disabled):hover::before{
     background-color:${colors.black};
   }
+  .sberRadioGroupIcon .ant-radio-button-wrapper {
```

Comment:
Is it possible to move it to radio group component?

Reply:
Done